starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,43 @@
+
+* {
+  font-family: 'Open Sans', sans-serif; 
+  font-size: 30px;
+}
+
+.header {
+  background-image: url(file:///home/looffi/code/labs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg);
```

Comment:
Using this path for the image will work on your computer, but not on others (not on mine for instance), and not when you push your projects online!
This is why your image paths should be "relative", meaning the path goes from your index.html file to your image file inside your folders. Here, the correct path would be: `'../images/apple-pie.jpg'`